### PR TITLE
Haskell code (using Scotty)

### PR DESCRIPTION
**Pull request by psibi**

### CODE REVIEW

**Comment by bgw** [comment]
My vote goes to this one over #63, as scotty is the more popular library, this implementation is shorter, and a cabal file is included.

**Comment by psibi** [comment]
@zpao Any updates on this ?

File: react-scotty.cabal [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-- Initial react-scotty.cabal generated by cabal init.  For further 
+-- documentation, see http://haskell.org/cabal/users-guide/
+
+name:                react-scotty
+version:             0.1.0.0
+synopsis:            React Haskell code with Scotty
+-- description:         
+license:             GPL-2
```
Comment (zpao) [inline react-scotty.cabal]: Could you

**Comment by zpao** [comment]
I can't get it to run but it looks pretty right so let's do it.

**Comment by psibi** [comment]
@zpao Thanks, I will also add instructions on how to install it using [stack](https://github.com/commercialhaskell/stack) which is a much easier.

**Pull request merged**